test/run/FshToFhir.test.ts

```diff
@@ -163,6 +163,7 @@ describe('#FshToFhir', () => {
       expect(results.fhir[0].id).toBe('MyPatient');
       const name = results.fhir[0].differential.element.find((e: any) => e.id == 'Patient.name');
       expect(name.mustSupport).toBe(true);
+      expect(results.fhir[0].snapshot).not.toBeDefined();
```

Comment:
This is nitpicky, and this code does not need to be changed, but for future consideration, Jest does support [expect(thing).toBeUndefined()](https://jestjs.io/docs/expect#tobeundefined).

Reply:
Somehow, I've missed that! I learn something new everyday.